wiki-webapp/src/main/webapp/skin/less/notes/notes.less

```diff
@@ -34,7 +34,86 @@
 	}
 }
 
-
+/************ New css feature notes ****************/
+#notesTop {
+	display: block;
+	box-shadow: 0 3px 5px hsla(0, 1%, 43%, 0.8);
+	z-index: 10;
+	.cke_top {
+		height: auto;
+	}
+
+	.cke_toolbox {
+		flex-flow: wrap;
+		background-color: #fafafa;
+		border-bottom: 0;
+	}
+
+	.cke_shared {
+		border: 0;
+	}
+}
+.formInputGroup{
+	display: flex;
+	margin: 0px 1%;
+	padding: 10px 0;
+	#cke_1_contents{
+		height: 400px;
+	}
+}
+#cke_notesContent {
+	border-color: transparent;
+	flex: 0 1 100%;
+	margin-bottom: 0;
+}
+#notesEditor{
+	background-color: #FFFFFF;
+	.notesActions{
+		.notesFormButtons{
+			display: inline-flex;
+			flex-wrap: wrap;
+			justify-content: space-between;
+			width: 100%;
+			padding: 10px;
+			margin: 0;
+			.notesFormLeftActions{
+				display: inline-flex;
+				flex-direction: row;
+				flex: auto;
+				justify-content: normal;
+				img {
+					height: 40px;
+				}
+				input[type="text"] {
+					flex: 0 1 100%;
+					margin-bottom: 0;
+					border-radius: 0px;
+					border-color: transparent;
+					padding: 0 20px;
+					font-size: 24px;
+				}
+			}
+			.notesFormRightActions{
+				display: inline-flex;
+				padding-right: 30px;
+				#notesUpdateAndPost{
+					height: 36px;
+					min-width: 64px;
+					padding: 0 16px;
+					color: #FFFFFF !important;
```

Comment:
Same remark

Reply:
Ad use Vuetify css helpers for spacing : https://vuetifyjs.com/en/styles/spacing/